plugins/splitwindow.c

```diff
@@ -318,10 +318,15 @@ static void split_view(gboolean horizontal)
 	gtk_container_add(GTK_CONTAINER(pane), notebook);
 	g_object_unref(notebook);
 
+	/* used just to get us to the same visual height level as the main editor window */
+	viewport = gtk_viewport_new(NULL, NULL);
```

Comment:
BTW, note that apparently some themes (at least Adwaita for GTK2) draw the GtkViewport's frame different than GtkFrame and GtkScrolledWindow's